src/pages/PrintOnlyScreen.tsx

```diff
@@ -50,7 +50,7 @@ const PrintOnlyScreen = ({
   const printerTimer = useRef(0)
   const [okToPrint, setOkToPrint] = useState(true)
   const [isPrinted, updateIsPrinted] = useState(false)
-  const [trackerString, setTrackerString] = useState<string | undefined>('')
+  const [trackerString, setTrackerString] = useState('')
```

Comment:
So I'm trying to wrap my head around this. The initial value of the tracking string is the empty-string, which gets set by printBallot, which is inside PrintOnlyScreen. I'm just trying to convince myself that the tracker is always set before the printing happens. That seems to bottom out to a call to the ElectionGuard server (/electionguard/EncryptBallot), which may or may not be idempotent (i.e., it might return something different each time and/or have other side-effects).

On the one hand, I'd say a little bit of paranoia is appropriate here, in the rare case that printing is triggered twice. Maybe you want to hang onto the result from the ElectionGuard server, but then you have to also make sure you throw it away once the user session is done.

(Or, maybe I'm just over-thinking things?)